nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -563,6 +563,12 @@ Misc projectile types, effects, think of this as the special FX file.
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
 
+/obj/item/projectile/beam/laser/phaser/pd
+	name = "point defense phaser"
+	damage = 80 // Doesn't scale with power input, but fires fairly quickly especially when upgraded
```

Comment:
Since this has a higher rate of fire than the ballistic gauss, it should do a bit less damage